tests/scenarios/package.json

```diff
@@ -91,23 +91,25 @@
     "ember-source-4.12": "npm:ember-source@~4.12.0",
     "ember-source-4.4": "npm:ember-source@~4.4.0",
     "ember-source-4.8": "npm:ember-source@~4.8.0",
+    "ember-source-5.12": "npm:ember-source@~5.12.0",
     "ember-source-5.4": "npm:ember-source@~5.4.0",
     "ember-source-5.8": "npm:ember-source@~5.8.0",
-    "ember-source-5.12": "npm:ember-source@~5.12.0",
     "ember-source-beta": "npm:ember-source@beta",
     "ember-source-canary": "npm:ember-source@alpha",
     "ember-source-latest": "npm:ember-source@latest",
+    "ember-template-imports": "^4.1.2",
     "ember-test-helpers-2": "npm:@ember/test-helpers@^2.0.0",
     "ember-truth-helpers": "^3.0.0",
     "ember-welcome-page": "^7.0.2",
-    "ember-template-imports": "^4.1.2",
     "execa": "^5.1.1",
     "node-fetch": "2.7.0",
     "popper.js": "^1.16.1",
     "strip-ansi": "^6.0.0",
     "testem": "^3.10.1",
     "tslib": "^2.6.0",
     "typescript": "^5.4.5",
+    "vite-5": "npm:vite@^5.0.0",
+    "vite-6": "https://pkg.pr.new/vite@main",
```

Comment:
We should probably wait for a beta to be released rather than keep this floating 🤔